backend/pom.xml

```diff
@@ -14,12 +14,17 @@
 
     <dependencies>
 
-        <!-- notifications-common module -->
+        <!-- notifications modules -->
         <dependency>
             <groupId>com.redhat.cloud.notifications</groupId>
             <artifactId>notifications-common</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.redhat.cloud.notifications</groupId>
+            <artifactId>notifications-database</artifactId>
```

Comment:
This dependency ensures that it's executed by the backend?

Reply:
mmh, I see it's on the engine and the aggregator.

Reply:
The dependency on `notifications-database` does not execute/trigger anything. It's what we do with Flyway that determines where and when the migration will happen.

Reply:
And that didn't change in this PR:
- notifications-aggregator triggers a Flyway migration at startup in test mode only
- notifications-backend triggers a migration at startup all the time
- notifications-engine does not trigger a migration yet, but will do it in test mode only soon